runtime/dart_isolate.cc

```diff
@@ -725,6 +725,31 @@ bool DartIsolate::MarkIsolateRunnable() {
   return true;
 }
 
+static void InvokeDartPluginRegistrantIfAvailable(Dart_Handle library_handle) {
+  TRACE_EVENT0("flutter", "InvokeDartPluginRegistrantIfAvailable");
+
+  // The Dart plugin registrant is a static with signature `void register()`
+  // within the class `_PluginRegistrant` generated by the Flutter tool.
+  //
+  // This method binds a plugin implementation to their platform
+  // interface based on the configuration of the app's pubpec.yaml, and the
+  // plugin's pubspec.yaml.
+  //
+  // Since this method may or may not be defined, check if the class is defined
+  // in the default library before calling the method.
+  Dart_Handle plugin_registrant =
+      ::Dart_GetClass(library_handle, tonic::ToDart("_PluginRegistrant"));
+
+  if (Dart_IsError(plugin_registrant)) {
+    return;
+  }
+  Dart_Handle result =
+      tonic::DartInvokeField(plugin_registrant, "register", {});
+  if (tonic::LogIfError(result)) {
+    FML_LOG(ERROR) << "The Dart plugin registrant produced an error";
+  }
```

Comment:
The extra logging here is probably not helpful - `LogIfError` will already emit some log information about what went wrong, and this line of logging will happen after that logging.

Reply:
Removed. I was following @chinmaygarde pattern for `_runMainZoned`, `_getStartMainIsolateFunction`, etc...